tests/modules/span_extractors/endpoint_span_extractor_test.py

```diff
@@ -17,7 +17,7 @@ def test_endpoint_span_extractor_can_build_from_params(self):
                 })
         extractor = SpanExtractor.from_params(params)
         assert isinstance(extractor, EndpointSpanExtractor)
-        assert extractor.get_output_dim() == 10
+        assert extractor.get_output_dim() == 17 # 2 * input_dim + span_width_embedding_dim
```

Comment:
Two spaces before `#` - makes it easier to see it's a comment, particularly with this github highlighting.